pages/common/gml2gv.md

```diff
@@ -0,0 +1,17 @@
+# gml2gv
+
+> Convert a graph from the `gml` format to the `gv` format.
+> Converters: `gml2gv`, `gv2gml`, `gv2gxl`, `gxl2gv`, `graphml2gv` & `mm2gv`.
+> More information: <http://graphviz.org/pdf/gml2gv.1.pdf>.
+
+- Convert a graph from the `gml` format to the `gv` format:
+
+`gml2gv -o output.gv input.gml`
+
+- Make same conversion using `stdin` and `stdout`:
```

Comment:
```suggestion
- Convert a graph using stdin and stdout:
```

Reply:
I had my doubts about "Make same conversion". I like "Convert a graph" better too.

I back-quoted stdin and stdout as because they are filenames. However "grep stdin *.md" says that you have not been quoting those. I'm discovering that there is much not covered by your style guide. I will unquote these and the other pending PRs that use `stdin` and `stdout`.

Reply:
I'm not exactly sure why we don't use `` `stdin` ``, but I think @sbrl mentioned something about it a while ago.
Also, do you think it would help new contributors if we covered every detail in our style guide or shuld we just leave some minor things up to the review process?

Reply:
A lawyer would tell you to write it all down and use the review process to interpret and enforce the rules. A king would tell you to leave it all up to what he thinks.